package.json

```diff
@@ -21,7 +21,7 @@
     "test:post": "jest",
     "prebuild": "cp ./main/react-native.config.cjs ./lib/react-native.config.cjs && rm -rf ./lib/tsconfig* && cp README.md ./lib/README.md ",
     "build": "yarn prebuild && tsc --project tsconfigBuild.json && yarn cp:icon",
-    "cp:icon": "mkdir -p `dirname ./lib/Icon/selection.json` && cp ./main/Icon/selection.json ./lib/Icon/selection.json && cp ./main/Icon/doobooui.ttf ./lib/Icon/doobooui.ttf",
+    "cp:icon": "mkdir -p `dirname ./lib/Icon/selection.json` && cp main/Icon/selection.json ./lib/Icon/selection.json && cp main/Icon/doobooui.ttf ./lib/Icon/doobooui.ttf",
```

Comment:
Why removing `./` only in `main`? You'd want to remove `./` in `lib` too and unify them.